src/analysis/modals/CloudEnvironmentModal.ts

```diff
@@ -312,6 +312,7 @@ export const CloudEnvironmentModal = ({
         );
       case appStatuses.starting.status:
       case appStatuses.stopping.status:
+      case appStatuses.updating.status:
```

Comment:
@jdcanas I was not quite sure if the default case would handle this correctly. Another potential area is here https://github.com/DataBiosphere/terra-ui/blob/1334689f57a6993a8534f52c1ff9145af34ac8c3/src/analysis/utils/app-utils.ts#L77

But in that case I think the default status will kick in and capitalize